lib/openstack_query/runners/project_runner.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, List
+import logging
+
+from openstack.identity.v3.project import Project
+from exceptions.parse_query_error import ParseQueryError
+
+from openstack_api.openstack_connection import OpenstackConnection
+from openstack_query.runners.runner_wrapper import RunnerWrapper
+
+from custom_types.openstack_query.aliases import ServerSideFilters
+
+logger = logging.getLogger(__name__)
+
+# pylint:disable=too-few-public-methods
+
+
+class ProjectRunner(RunnerWrapper):
+    """
+    Runner class for openstack flavor resource
+    ProjectRunner encapsulates running any openstacksdk Project commands
+    """
+
+    RESOURCE_TYPE = Project
+
+    def _parse_meta_params(self, _: OpenstackConnection, **__):
+        logger.error(
+            "ProjectQuery doesn't take any meta-params, if you think it should,"
+            "please raise an issue with the repo maintainers"
+        )
```

Comment:
I would drop this error, `FlavorQuery has no meta-params available` is pretty descriptive enough tbh